tools/flaky-test-retryer/github_commenter.go

```diff
@@ -158,19 +170,28 @@ func buildNewComment(jd *JobData, entries map[string]int, outliers []string) str
 	}
 	sort.Strings(keys)
 	for _, test := range keys {
-		entryString = append(entryString, fmt.Sprintf("%s | %d/%d", test, entries[test], maxRetries))
+		entryString = append(entryString, fmt.Sprintf("%s | %s | %d/%d", test, buildLinks(entries[test].oldLinks, jd.GCSPath), entries[test].retries, maxRetries))
```

Comment:
you can also get the build-log url from the reportmessage directly. We dont need to replace gcs with spyglass

Reply:
cool, didn't even notice before